### PR TITLE
add responsive media query and card component in grid

### PR DESCRIPTION
**Pull request by priyalpepcus**

### CODE REVIEW

File: src/card-print.css [hunk 1]
```diff
@@ -0,0 +1,187 @@
+@media print {
```
Comment (pepcus-pratik) [inline src/card-print.css]: Isn't this style is getting applied on all pages?

File: src/components/index [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<!doctype html>
```
Comment (pepcus-pratik) [inline src/components/index]: Looks like a unnecessary file

File: src/dataGrid.css [hunk 3]
```diff
@@ -856,22 +820,24 @@ html, body, #root, #root>div, .student-information-Container {
...
     z-index:2;
+    padding:0px 0;
+    background: #f9570a;
     color: #fff;
     font-size: 26px;
     width: 100%;
     text-align: center;
+    left: 0;
+}
+.student-information-Container h2 {
```
Comment (pepcus-pratik) [inline src/dataGrid.css]: Targetting html tags inside a container is not a good practice

File: src/card-print.css (hunk 1 above)
Comment (priyalpepcus) [reply src/card-print.css]: No, this style is applied on only grid page card component.

File: src/components/index (hunk 2 above)
Comment (priyalpepcus) [reply src/components/index]: file removed

**Pull request merged**